src/dlgConnectionProfiles.cpp

```diff
@@ -1760,11 +1739,11 @@ bool dlgConnectionProfiles::validateProfile()
             if (!allowedChars.contains(name.at(i))) {
                 notificationAreaIconLabelWarning->show();
                 notificationAreaMessageBox->setText(
-                        QString("%1\n%2").arg(notificationAreaMessageBox->text(), tr("The %1 character is not permitted. Use one of the following:\n\"%2\".\n").arg(name.at(i), allowedChars)));
+                        QString("%1\n%2").arg(notificationAreaMessageBox->text(), tr("The %1 character is not permitted. Use one of the following:\"%2\".").arg(name.at(i), allowedChars)));
                 name.replace(name.at(i--), QString());
+                profile_name_entry->setPalette(mErrorPalette);
```

Comment:
:bulb: This is a separate matter but we should be moving away from using `palette` functions and switching to `stylesheet` ones as they do not play nicely together according to Qt and we are seeing more and more usage of application styling generally.